### ISSUE TITLE
Ngdbuild failed with two errors because of CLKIN2_PERIOD settings

### ISSUE THREAD

**Issue by kartharer** [parallella/parallella-hw#7]
Following the Readme in fpga/projects/parallella_7020_headless on a Ubuntu 14.04 with PlanAhead 14.7 the bitstream generation failed with 14 critical warnings and two errors.

First I got:

[Netlist 29-73] Incorrect value '10.000000' specified for property 'CLKIN1_PERIOD'. The system will either use the default value or the property value will be dropped. Verify your source files. ["/home/marvin/Development/parallella-hw/fpga/projects/parallella_7020_headless/parallella_7020_headless.runs/synth_1/parallella_z7_top.ngc":70359]

and for CLKIN2_PERIOD the same but with the value '0.000000'. These both result in the following error for the ngdbuild task:

ERROR:LIT:374 - Attribute CLKIN1_PERIOD on MMCME2_ADV instance
   "parallella/ewrapper_link_top/io_clock_gen/mmcm_adv_inst" has invalid value
   "0,000000". The CLKIN1_PERIOD attribute should have a real number, followed
   by optional time or frequency units; nS are assumed if no units are given.

The same error will occur also for CLKIN2_PERIOD. This point is interesting because in io_clock_gen_600mhz.v is only a definition for CLKIN1_PERIOD and no definition for CLKIN2_PERIOD. Also the value set for CLKIN1_PERIOD with '(10.0)' looks correct (according to the Xilinx documentation).

As you can see the comment for CORE_GENERATION_INFO in this file, there was an earlier version where a CLKIN2_PERIOD with value 10.0.

Searching for CLKIN2_ in an clean git clone of the projects found some files, interestingly none with the value 0.0. According to the Xilinx docu xco-files are only logfiles and should not parsed by the build process.

The other files that contains CLKIN2_ come from external IP cores in the folder /fpga/externals/fpgahdl_xilinx/motor_control/. I assume that these files are not relevant for the project, or?

I've tried to delete the xco-files with no effect. Another try to add the definition of CLKIN2_PERIOD also has no effect, still same errors occur.

In conclusion it looks as some definitions of the zynq in the Xilinx toolchain differ from others. So while the implementation processes run the settings in the project will be overwritten.
